docs/user-guide/standard_transformer_upi.md

```diff
@@ -0,0 +1,99 @@
+# Configuring Standard Transformer for UPI Model
+
+> This guide assumes you have experience using standard transformer and  familiar with UPI contract. You can read refer to https://github.com/caraml-dev/universal-prediction-interface to get details on the contract. 
+
+There are 2 key differences in Standard Transformer when it’s deployed using UPI protocol:
+
+1. Autoload Feature
+2. Separate Output Operation for Pre-process and Post-process
+
+## Autoload Feature
+
+Autoload feature is the primary mechanism for importing values in the request payload as variables or tables into standard transformer. Previously, it is done by using JSONPath query in the HTTP mode.
+
+For example in HTTP model, if you want to declare a rating variable that should use value from user_rating field of the incoming request, then you have to declare following configuration in the input.
+
+```yaml
+  - variables:
+      - name: rating
+        jsonPathConfig: 
+          jsonPath: $.user_rating
+          defaultValue: -1
+          valueType: INT
+```
+
+You can avoid it altogether by using autoload feature in UPI. To do so:
+
+- Store the variable in `transformer_input` field of the `PredictValuesRequest` 
+
+For example, when using Python SDK, you can do so by following code.
+
+```python
+from caraml.upi.v1 import type_pb2, upi_pb2_grpc, upi_pb2, variable_pb2
+
+request = upi_pb2.`PredictValuesRequest`(
+    ...
+    transformer_input=upi_pb2.TransformerInput(
+        variables=[
+            variable_pb2.Variable(name="user_rating", 
+                type=type_pb2.TYPE_DOUBLE, 
```

Comment:
In HTTP's spec above, the user_rating value type is INT, but it's double here. Is it expected?

Reply:
Updated